modules/serverless-sinkbinding-kn.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// serverless/knstive_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-kn_{context}"]
+= Using SinkBinding with the Knative CLI (`kn`)
+
+This guide describes the steps required to create, manage and delete a SinkBinding instance using `kn` commands.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have the `kn` CLI installed.
+
+.Procedure
+
+. To check that SinkBinding is set up correctly, create a Knative event display service, or event sink, that dumps incoming messages to its log.
++
+You can create the service by entering the following command:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+----
+
+. Create a SinkBinding that will direct events to the service.
++
+You can create the SinkBinding by entering the following command:
++
+----
+$ kn source binding create bind-heartbeat --subject Job:batch/v1:app=heartbeat-cron --sink svc:event-display
+----
+
+. Create a CronJob.
++
+You can do this by creating a file named `heartbeats-cronjob.yaml` and copying the following sample code into it:
```

Comment:
Maybe?
.Create a CronJob
.. Create a file named...
.. Apply the `heartbeats-cronjob.yaml` file:

Reply:
Must the YAML file be called `heartbeats-cronjob.yaml`?